### DISCUSSION TITLE
Bro the Documentation is not working!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by rajeshrah22** [JustFly1984/react-google-maps-api#3324] [General]
Bro the documentation is not working!!

**Comment by JustFly1984** [comment]
I know, I shut it down. Look directly into md files for docs, also
changelog.md

**Comment by kevinjavitz** [comment]
how much is the hosting cost?

**Comment by JustFly1984** [comment]
hosting is free, it is not the cost, it is a sequence of misfortunes.
1. I've build docs with react-styleguidist.
2. at some point react released react 18. 
3. react-styleguidist became unmaintained for a long time, and was incompatible with react@18 and build crashed, so I've stopped deploying, but have not removed docs from netlify.
4. past long time and somebody complained about serious bug in docs. I had to shut down docs website.
5. Somebody provided fix for the bug, but not the fix for the docs.
6. Docs build is still crashing, cos it requires updating react-styleguidist config and docs themselves.
7. Google has contacted me 2 times, but no sponsorship from them.
8. It is opensource. You can fix docs, send a PR and I will publish it to netlify again.
